plugins/integration-openai/.amplicationrc.json

```diff
@@ -0,0 +1,6 @@
+{
+  "settings": {},
+  "systemSettings": {
+    "requireAuthenticationEntity": "true"
```

Comment:
Why do you need the Authentication entity as mandatory for this plugin? I didn't see you reference it in the plugin code.